package/boot/uboot-mvebu/Makefile

```
@@ -29,6 +29,23 @@ define U-Boot/clearfog
   UBOOT_IMAGE:=u-boot-with-spl.kwb
 endef
 
+define U-Boot/clearfog-emmc
+  NAME:=SolidRun ClearFog A1
+  BUILD_DEVICES:=solidrun_clearfog-base-a1 solidrun_clearfog-pro-a1
+  BUILD_SUBTARGET:=cortexa9
+  UBOOT_CUSTOMIZE_CONFIG:= \
+    --set-val SYS_MMCSD_RAW_MODE_U_BOOT_DATA_PART_OFFSET 0x1000
+  UBOOT_IMAGE:=u-boot-with-spl.kwb
+endef
+
+define U-Boot/clearfog-sata
+  NAME:=SolidRun ClearFog A1
```

They all have the same name in menuconfig. Please add a postfix like:
```suggestion
  NAME:=SolidRun ClearFog A1 (SATA)
```